src/aotcompile.cpp

```diff
@@ -514,7 +506,8 @@ static void injectCRTAlias(Module &M, StringRef name, StringRef alias, FunctionT
     if (!target) {
         target = Function::Create(FT, Function::ExternalLinkage, alias, M);
     }
-    Function *interposer = Function::Create(FT, Function::InternalLinkage, name, M);
+    Function *interposer = Function::Create(FT, Function::ExternalLinkage, name, M);
```

Comment:
Hm not sure about this change... Needs to be only present until after machine generation and we really don't want anyone else to link against it.

Reply:
In that case would RAUW work instead of injecting the interposer?

Reply:
bump. It'd be nice to get this merged